pkg/shell/credentials.js

```diff
@@ -30,14 +30,17 @@ define([
     function Keys() {
         var self = this;
 
-        self.path = cockpit.user["home"] + "/.ssh";
+        self.path = null;
         self.items = { };
 
         var watch = null;
         var proc = null;
         var timeout = null;
 
-        refresh();
+        cockpit.user().done(function (user) {
+            self.path = user.home + '/.ssh';
+            refresh();
+        });
```

Comment:
There's a risk that self.path won't be set by the time various things that need it are called. However this is better than the status quo, where it'll be and stay invalid in that case.